packages/venia-ui/lib/components/ShoppingBag/shoppingBag.css

```diff
@@ -42,10 +42,8 @@
     gap: 1rem;
     /* The header and footer heights may change based on PWA-116 and PWA-601. */
     grid-template-rows: 2rem 1fr minmax(2rem, auto);
-    /* These heights may change based on PWA-605 and should be based on 1.5 and 2.5 visible items. */
-    height: 100%;
-    min-height: 31rem;
-    max-height: 45rem;
+    min-height: 10rem;
```

Comment:
I have calculated the height for each item in the shopping bag and for 2.5 items it is 32rem. Hence the `max-height` is 32rem. I am not sure if that is the right way to handle this, I am open to suggestions.